docs(image): Add section for adding new packages (#3575)
edgelesssys/constellation: image/README.md

  - `bazel build //image/system:IMAGE_NAME_HERE` (replace with an actual image name)
- Let CI build new images and run e2e tests
- Upgrade kernel spec under [edgelesssys/constellation-kernel](https://github.com/edgelesssys/constellation-kernel) to use new releasever

## Adding new packages to the image

> - Add your package to the corresponding section in `./base/mkosi.conf`

Reviewer: ```suggestion
- Find the package (i.e. it's _package name_) on [Koji](https://koji.fedoraproject.org/koji/)
- Add the package to the corresponding section in `./base/mkosi.conf`
  - If the package is required to be present in the initrd, add it to `./initrd/mkosi.conf`
```

> - Add your package to `./mirror/packages.txt`

Reviewer: ```suggestion
- Add the package to `./mirror/packages.txt`
```

- Update the package mirror:
  ```sh
  bazel run //image/mirror:update_packages
  ```

> - Let CI build new images and run e2e tests

Reviewer: ```suggestion
- Build new images (e.g. via CI) and run e2e tests
```
